src/osd/CMakeLists.txt

```diff
@@ -46,7 +46,7 @@ endif()
 add_library(osd STATIC ${osd_srcs})
 target_link_libraries(osd
   PUBLIC dmclock::dmclock
-  PRIVATE os heap_profiler cpu_profiler ${CMAKE_DL_LIBS})
+  PRIVATE os heap_profiler cpu_profiler ceph-common ${CMAKE_DL_LIBS})
```

Comment:
Same here for the `ceph-common`

Reply:
It's about the same issue, but it mainly affects Windows, which doesn't allow unresolved dependencies.

Reply:
this breaks the build on bionic. 
```
dpkg-shlibdeps: error: cannot find library libceph-common.so.2 needed by debian/ceph-osd/usr/bin/ceph-osd (ELF format: 'elf64-x86-64' abi: '0201003e00000000'; RPATH: '')
```

see https://jenkins.ceph.com/job/ceph-dev-new-build/ARCH=x86_64,AVAILABLE_ARCH=x86_64,AVAILABLE_DIST=bionic,DIST=bionic,MACHINE_SIZE=gigantic/39412//consoleFull